Mathlib/Topology/CWComplex/Classical/Basic.lean

```diff
@@ -571,6 +599,20 @@ lemma CWComplex.iUnion_openCell_eq_skeleton [CWComplex C] (n : ℕ∞) :
     ⋃ (m : ℕ) (_ : m < n + 1) (j : cell C m), openCell m j = skeleton C n :=
   iUnion_openCell_eq_skeletonLT _
 
+lemma RelCWComplex.iUnion_skeletonLT_eq_complex [RelCWComplex C D] :
+    ⋃ (n : ℕ), skeletonLT C n = C := by
+  apply subset_antisymm (iUnion_subset_iff.2 fun _ ↦ skeletonLT_subset_complex)
+  simp_rw [← union_iUnion_openCell_eq_complex , union_subset_iff, iUnion₂_subset_iff]
+  exact ⟨subset_iUnion_of_subset 0 (base_subset_skeletonLT ↑0),
+    fun n i ↦ subset_iUnion_of_subset _ (openCell_subset_skeletonLT n i)⟩
+
+lemma RelCWComplex.iUnion_skeleton_eq_complex [RelCWComplex C D] :
+    ⋃ (n : ℕ), skeleton C n = C := by
+  apply subset_antisymm (iUnion_subset_iff.2 fun _ ↦ skeleton_subset_complex)
```

Comment:
This is also exactly the same proof as above. Is there a way to unify these two proofs?